docs/features/role-based-access-control/brief.md

```diff
@@ -0,0 +1,40 @@
+# Role base access control - Brief
+
+This document aims at providing an overview of Role-based Access Control (RBAC from here on) for the Marxan platform. 
+
+In broad strokes, RBAC assumes the existence of a set of roles, which are associated with different actions that they 
+can perform in the platform. Users can then be associated with roles for specific resources, which will grant them the 
+role privileges for that specific resource.
+
+The current implementation is based on the requirements described in the [User requirements document](https://docs.google.com/document/d/16x4wQYTKov0JkO1ue6iQzJ3-RXpdgQ38ygGJWW86Zo0/edit#heading=h.58xbt2es73r2).
+
+## Roles
+
+In the context of the Marxan platform, we will consider the following roles:
+
+* **Admin**: a generic platform administrator, who can perform **any action for any resource** in the platform.
```

Comment:
I recommend to split this role from the others, to avoid confusion as it will apply only to the platform as a whole rather than to specific resources such as orgs/projects/scenarios. The current explanation says so, but I'd even treat it as part of a different list, for clarity.

Reply:
Also, I recommend to make this role orthogonal to the other roles. The working document may not be fully clear on this, but my recommendation is that the concerns (and access rights) of a platform admin should be mostly distinct from those of entity owners/contributors/viewers/etc.

In other words, I would recommend *not* to make this a "root" role but a role with given permissions on the platform but not implicit nor explicit permissions on entities, especially projects and scenarios, *except* for being able to see title/description of projects and potentially delete them, but not necessarily being able to see project content and content of scenarios within projects.

This is partly a matter of personal preference from past experience (playing the greybeard card here 😜) but also from conversations with TNC, I know that there may be concerns about "root" users (besides the unavoidable ones such as infra admins)

Reply:
Done and done :+1: